src/app/components/case/forms/CitizenReportForm/scaffold.tsx

```diff
@@ -41,9 +41,10 @@ export default (caseId: Components.Schemas.Case["id"]) => {
           type: "TelField",
           props: {
             label: "Telefoonnummer melder",
+            extraLabel: "(indien bekend)",
             name: "reporter_phone",
             hint: "Vul hier alleen cijfers in",
-            isRequired: true,
+            isRequired: false,
             validate: (value: string) => (value === undefined || /^[0-9]{10}$/.test(value.trim())) ? false : "Vul hier enkel 10 cijfers in"
```

Comment:
I think `/^[0-9]{10}$/.test(value.trim()) ? false : "Vul hier enkel 10 cijfers in"` would also work when `isRequired: true`

Reply:
The field is no longer required. But if someone enters a value, it has to be 10 characters long.

Reply:
Ah, Ok. Read that wrong, then.

Reply:
Maybe update the param type with undefined `validate: (value: string | undefined) => `